### PR TITLE
readme: add Communication section in readme

### PR DESCRIPTION
**Pull request by dcalvin**
Signed-off-by: Calvin Weng <wenghao@pingcap.com>

## What have you changed? (mandatory)

- Added Communication section in readme as suggested by Siddon

## What are the type of the changes? (mandatory)

- Improvement

### CODE REVIEW

**Comment by dcalvin** [comment]
Anything else to add？

File: README.md [hunk 1]
```diff
@@ -181,6 +181,32 @@ The TiKV team actively develops and maintains a bunch of dependencies used in Ti
...
+- [Reddit](https://www.reddit.com/r/TiKV)
+- Post questions or help answer them on [Stack Overflow](https://stackoverflow.com/questions/tagged/tikv)
+
+### Slack
+
+Join TiKV community on [Slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LTgzZDQ3NzZlNDkzMGIyYjU1MTA0NzIwMjFjODFiZjA0YjFmYmQyOTZiNzNkNzg1N2U1MDdlZTIxNTU5NWNhNjk) - Sign up and join channels on TiKV topics that interest you. 
+
+### WeChat
+
+TiKV community is also available on WeChat, a very popular channel for instant messages and discussions in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personel information (WeChat ID, and other basic information) included. We will invite you in right away.
```
Comment (Hoverbear) [inline README.md]:
```suggestion
TiKV community is also available on WeChat, a very popular channel for instant messages and discussions in China. If you want to join our WeChat group, send a request mail to [tl@pingcap.com], with your personal information (WeChat ID, and other basic information) included. We will invite you in right away.
```

File: README.md [hunk 2]
```diff
@@ -181,6 +181,32 @@ The TiKV team actively develops and maintains a bunch of dependencies used in Ti
...
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+### Socia Media
+
+- [Twitter](https://twitter.com/tikvproject)
+- [Blog](https://tikv.org/blog/)
+- [Reddit](https://www.reddit.com/r/TiKV)
+- Post questions or help answer them on [Stack Overflow](https://stackoverflow.com/questions/tagged/tikv)
```
Comment (Hoverbear) [inline README.md]: I'm still working on the DNS for discourse. Q_Q

**Comment by brson** [comment]
Awesome awesome stuff @dcalvin 🥇 LGTM

**Pull request merged**